autogen/sr.rs

```diff
@@ -243,6 +248,12 @@ pub fn gen_sr_code_from_instruction_grammar(
         // Get the token for its enumerant
         let inst_name = &inst.opname[2..];
         let name_ident = Ident::new(inst_name, Span::call_site());
+        let type_name = if inst.opname.len() > 6 {
```

Comment:
What is this mysterious magic number here? Could you add some comment to explain? Thanks!